web/app/stylesheet.css.sass

```diff
@@ -1,4 +1,109 @@
 @import "bootstrap"
 
-body
-  margin: 10px 0
+a
+  font-weight: 400
+  font-family: 'open_sans_regular', 'Helvetica Neue', Helvetica, Arial, sans-serif
+  text-decoration: none
+  list-styling: none
+  color: white
+  font-size: 15px
+  padding: 20px
+  letter-spacing: 0.5px
+
+a:hover
+  font-weight: 400
+  font-family: 'open_sans_regular', 'Helvetica Neue', Helvetica, Arial, sans-serif
+  text-decoration: none
+  list-styling: none
+  color: white
+
+h2
+  margin-bottom: 0
+  margin-left: 10%
+  padding: 10px
+  color: white
+  height: 55px
+  font-size: 17px
+  display: flex
+  align-items: center
+  letter-spacing: 0.5px
+  font-weight: 400
+  font-family: 'open_sans_regular', 'Helvetica Neue', Helvetica, Arial, sans-serif
+
+.navbar
+  padding: 0
+  background-color: rgba(14, 34, 52, 1)
+  height: 77px
+
+.cyber_logo
+  width: 205px
+  margin: 30px
+
+.nav-item
+  font-size: 20px
+  display: flex
+  align-items: center
+
+.btn
+  border-radius: 5px
+  font-family: 'open_sans_regular', 'Helvetica Neue', Helvetica, Arial, sans-serif
+  font-size: 14px
+
+.btn_default
+  color: #444
+  border-color: #ddd
+  border: 1px solid transparent
+  border-radius: 2px
+
+.dropdown-item
+  font-family: 'open_sans_regular', 'Helvetica Neue', Helvetica, Arial, sans-serif
+  font-size: 14px
+
+.wrapper-policy
+  background-color: rgba(14, 34, 52, 1)
+
+.policy-banner 
+  background-color: rgba(77, 132, 184, 1)
+  margin-bottom: 0
+
+.btn-secondary
+  background-color: rgba(77, 132, 184, 1)
+  border: 0
+  height: 47px
+  border-radius: 5px
+
+.btn-secondary:hover
+  background-color: #224A6F
+
+.policy-temp-wrapper
+  display: flex
+  align-items: center
+
+.input-group-text
+  background-color: #0e2234
+  color: white
+  letter-spacing: 0.5px
+  font-weight: 400
+  font-family: 'open_sans_regular', 'Helvetica Neue', Helvetica, Arial, sans-serif
+  font-size: 15px
+  border: 0
+
+.form-control
+  font-size: 15px
+
+.policy-addition-sidebar
+  position: fixed
+  bottom: 0px
+  left: 8px
+  right: 0px
+  top: 145px
+  width: 17%
+  height: 28%
+
+@media only screen and (max-width: 1700px) 
+  .policy-addition-sidebar
+    position: relative
+    top: 0px
+    width: 100%
+
+    
```

Comment:
I'm not sure what this is trying to accomplish. `position: relative` means we're supplying an offset from the normal computed position. (https://developer.mozilla.org/en-US/docs/Web/CSS/position)

Is this intended to give an offset of zero?

Overall, how would you describe the strategy here?

Reply:
it looks like this was trying to keep the buttons close to the "Download" button. If we used bootstrap grid classes (.col-md-6) or something like css grid or flexbox, we wouldn't need the positioning on the columns. I tried to play w the page to get this working, but the Ruby is greek to me.

Reply:
@sigalsax I pushed a couple of tiny css tweaks up to the PR (the biggest being using .scss instead of .sass) but I think we need to do some template re-work to get the overall html structure we want, something like:
.navbar
.page-heading
.row
  .sidebar
  .policy-wrapper
    .policy
    .download-policy-btn